TLM/TLM/U/Autosize/UPadding.cs

```diff
@@ -0,0 +1,38 @@
+namespace TrafficManager.U.Autosize {
+    public struct UPadding {
+        public float Top;
+        public float Right;
+        public float Bottom;
+        public float Left;
+
+        public static UPadding Zero() {
+            return new() {
+                Right = 0f,
+                Top = 0f,
+                Left = 0f,
+                Bottom = 0f,
+            };
+        }
+
+        /// <summary>Set padding on all sides to <see cref="UConst.UIPADDING"/> (4f).</summary>
+        public static UPadding Const() {
+            return SameValue(UConst.UIPADDING);
+        }
+
+        public static UPadding SameValue(float v) {
```

Comment:
why is this method necessary? strucs are copied by value anyway.
if this was a class I can understand the need for a clone function but why do we need this for such a simple struct?

Reply:
This is not a method, is a constructor.
This is only used once but i believe the future code will use it a bit more.
Probably this can be rewritten with a const struct, will that be any better?

Reply:
Replaced with 2 constant structs of 0 and UConst.UIPADDING.